docs/scenarios_authoring/Fragments.md

```diff
@@ -9,14 +9,34 @@ Fragments are a way to share processing logic - if many scenarios require identi
 Once a fragment is created using Designer (see Fragments tab) it can be used in other scenarios in the same category.
 
 ## Inputs
-Fragment can have one input. You can define parameters of a fragment:
+Fragment must have exactly one input. You can define parameters of a fragment:
 
-![fragment input](../autoScreenshotChangeDocs/Auto_Screenshot_Change_Docs_-_fragments_-_Inputs0.png)
+![fragment input_base](img/fragment_input_simple.png)
+
+Clicking the "Show advanced configuration" button opens up an additional form, that allows changing the parameter's behaviour during inputting its value when using the fragment, e.g. changing its validation requirements or displaying a hint.
+
+For types other than `String` and `Boolean` this allows configuration as seen here:
+
+![fragment input_advanced_long](img/fragment_input_advanced_long.png)
+
+While for parameters of type `String` or `Boolean`, on top of that it also allows choosing an input mode from `Any`, `Any with suggestions`, `Fixed list` and, if applicable, setting the items of the list.
```

Comment:
`while [..] , on top of` - not clear to me what we want to say here. Additionally, the type (?) 'any', 'any with suggestions', 'fixed list' does not show up [ihere](https://staging.nussknacker.io/visualization/amt_frag?nodeId=input)